packages/experimental/Button/src/Button.tsx

```diff
@@ -47,10 +47,23 @@ const ButtonComposed = compose<ButtonType>({
     const Slots = useSlots(userProps, (layer) => buttonLookup(layer, button.state, userProps));
 
     // now return the handler for finishing render
-    return (final: ButtonPropsWithInnerRef, ...children: React.ReactNode[]) => {
-      const { icon, iconPosition, loading, ...mergedProps } = mergeProps(button.props, final);
+    return (final: ButtonPropsWithInnerRef, children: React.ReactNode[]) => {
+      const { icon, iconPosition, loading, accessiblityLabel, ...mergedProps } = mergeProps(button.props, final);
+      let childText = '';
+      if (accessiblityLabel === undefined) {
+        React.Children.forEach(children, (child) => {
+          if (typeof child === 'string') {
+            if (childText.length !== 0) {
+              childText += ' ';
```

Comment:
Not sure if this raises i18n concerns... On the other hand, I think our stance for Buttons with more than one child is that you're kind of going into "Wild West" territory and we can't guarantee good experiences with that

Reply:
Would this not add a space even if there is one child?
I would rather Wild West territory be that we only support one string, rather than try to handle more than one string incorrectly

Reply:
Uh yep. That's supposed to be 1. Thanks